### PR TITLE
Removed "-common" packages

### PR DESCRIPTION
**Pull request by aderende**
## Problem

We need to reduce amount of npm packages for publishing YAML/JSON LSP server.

## Solution

Removed "-common" packages.
+ made a small fix for "fileMatch" field

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: server/aws-lsp-cloudformation/package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     },
     "dependencies": {
         "@aws/lsp-core": "^0.0.1",
-        "@aws/lsp-json-common": "^0.0.1",
-        "@aws/lsp-yaml-common": "^0.0.1",
+        "@aws/aws-lsp-yaml-json": "^0.0.1",
+        "@aws/aws-lsp-yaml-json": "^0.0.1",
```
Comment (viktorsaws) [inline server/aws-lsp-cloudformation/package.json:12]: nit: `@aws/aws-lsp-yaml-json` dependency is declared twice

File: server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export class JsonLanguageService implements AwsLanguageService {
             schemaRequestService: props.uriResolver?.bind(this),
         })
 
-        const schemas = props.defaultSchemaUri ? [{ fileMatch: ['*.json'], uri: props.defaultSchemaUri }] : undefined
+        const schemas = props.defaultSchemaUri ? [{ fileMatch: ['*'], uri: props.defaultSchemaUri }] : undefined
```
Comment (viktorsaws) [inline server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts:24]: Have you intentionally changed the extension in `fileMatch` field? What's the idea for any extension, should client define for which file extensions this server will be used?

File: server/aws-lsp-yaml-json/README.md
Comment (viktorsaws) [inline server/aws-lsp-yaml-json/README.md:1]: Could you update the documentation for this package to mention all components available in it?

File: server/aws-lsp-yaml-json/src/language-service/yamlLanguageService.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export class YamlLanguageService implements AwsLanguageService {
             workspaceContext,
         })
 
-        this.yamlService.configure({ schemas: [{ fileMatch: ['*.yml'], uri: this.props.defaultSchemaUri }] })
+        this.yamlService.configure({ schemas: [{ fileMatch: ['*'], uri: this.props.defaultSchemaUri }] })
```
Comment (viktorsaws) [inline server/aws-lsp-yaml-json/src/language-service/yamlLanguageService.ts:32]: Same question for extension, what's the intention for allowing any file extension?

File: server/aws-lsp-yaml-json/README.md
Comment (aderende) [reply server/aws-lsp-yaml-json/README.md:1]: sure, I can mention it

File: server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts (hunk 2 above)
Comment (aderende) [reply server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts:24]: discussed this comment in slack, this change is ok

File: server/aws-lsp-yaml-json/src/language-service/yamlLanguageService.ts (hunk 3 above)
Comment (aderende) [reply server/aws-lsp-yaml-json/src/language-service/yamlLanguageService.ts:32]: discussed this comment in slack, this change is ok

**Comment by justinmk3** [comment]
Why did `package-lock.json` increase by 8k lines?

> We need to reduce amount of npm packages for publishing YAML/JSON LSP server.

What exactly does this mean? What problem is being solved?

File: server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts (hunk 2 above)
Comment (imykhai) [reply server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts:24]: @aderende Could you also share the reasoning here, please?

File: server/aws-lsp-yaml-json/README.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+## Exports
+── index
+    └── CreateYamlJsonLanguageServer - a function that creates and returns a new instance of DEXP Language Server
```
Comment (imykhai) [inline server/aws-lsp-yaml-json/README.md:3]: Have it already been decided whether we distribute the json and yaml servers as one combined package?

**Review by imykhai — APPROVED** [review]

File: server/aws-lsp-yaml-json/README.md (hunk 4 above)
Comment (aderende) [reply server/aws-lsp-yaml-json/README.md:3]: not yet, but based on my performance testing I think it's not necessary https://quip-amazon.com/4AluAbChYCdz/LSP-testing-Analysis-and-results

File: server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts (hunk 2 above)
Comment (aderende) [reply server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts:24]: I replaced `fileMatch: ['*.yaml']` with `fileMatch: ['*']` every file’s URI in internal destination new API is "*" and it doesn't match "*.yaml" pattern.

**Pull request merged**